### PR TITLE
*: reformat code

### PR DESCRIPTION
**Pull request by huachaohuang**
Just reformat code without other changes.

### CODE REVIEW

**Comment by Hoverbear** [comment]
Can you make the CI run rustfmt? Like so: https://github.com/pingcap/raft-rs/blob/f271cd0dae4c868b2d92e19fe6c14e000598ea76/.travis.yml#L15-L19

**Comment by zhangjinpeng1987** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +15,13 @@ os:
 
 matrix:
   include:
-    - os: osx
-      rust: stable
+  - os: osx
+    rust: stable
+  - rust: nightly-2018-04-06
```
Comment (Hoverbear) [inline .travis.yml]: This version is used by TiKV but `rust-rocksdb` doesn't have this same requirement. I am worried later when we upgrade the version used by TiKV it means we need to change this. Perhaps it is better to do it on `stable`?

**Comment by DorianZheng** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (huachaohuang) [reply .travis.yml]: OK, let me try it on stable.

**Pull request merged**